decidim-core/app/views/decidim/links/new.js.erb

```diff
@@ -1,8 +1,24 @@
 (function() {
-  var $foo = $("<div />")
-  $foo.html('<%= j(render partial: "modal").strip.html_safe %>');
-  $("body").append($foo)
-  var $reveal = $(".reveal", $foo);
-  $foo.foundation()
-  $reveal.foundation("open")
+  const create = (selector) => {
+    const element = document.createElement("div")
+    element.id = selector
+    document.body.append(element)
+    return element
+  }
+
+  const selector = "external-domain-warning"
+  const selectorContainer = `${selector}-container`
+
+  // if the container does not exist in the DOM, it creates a new one, otherwise, replace the content
+  const externalDomainWarning = document.getElementById(selectorContainer) || create(selectorContainer)
+
+  externalDomainWarning.innerHTML = ''
```

Comment:
this first assingment is not necessary

Reply:
Enforces to clean the markup before the reassignment. This script could be called several times in a short term